src/encord_active/lib/common/filtering.py

```diff
@@ -55,7 +73,10 @@ def apply_filters(df: pd.DataFrame, filters: Filters):
     for column, text_filter in filters.text.items():
         filtered = filtered[filtered[column].astype(str).str.contains(text_filter)]
 
-    filtered.drop(columns=["data_row_id", "is_label_metric"], inplace=True)
+    columns_to_delete = ["data_row_id", "is_label_metric"]
+    if "label_hash" in filtered.columns:
+        columns_to_delete.append("label_hash")
+    filtered.drop(columns=columns_to_delete, inplace=True)
```

Comment:
if the suggestion above works this will not be necessary

```suggestion
    filtered.drop(columns=["data_row_id", "is_label_metric"], inplace=True)
```